src/App/App.xaml.cs

```diff
@@ -156,6 +155,10 @@ public App(AppOptions appOptions)
                             CheckPasswordlessLoginRequestsAsync().Wait();
                         }
                     }
+                    else if (message.Command == "syncCompleted")
+                    {
+                        await SyncPasswordlessLoginRequestsAsync();
+                    }
```

Comment:
I think this should be directly placed on the `SyncService` so everything related to syncing stuff is there and also the `App.xaml.cs` should not be the place to do this logic

Reply:
Agreed will move it there.